openc3/lib/openc3/models/interface_model.rb

```diff
@@ -242,6 +256,14 @@ def handle_config(parser, keyword, parameters)
         parser.verify_num_parameters(0, 0, "#{keyword}")
         @log_raw = true
 
+      when 'SECRET'
+        parser.verify_num_parameters(3, 4, "#{keyword} <Secret Type: ENV or FILE> <Secret Name> <Environment Variable Name or File Path> <Option Name (Optional)>")
+        @secrets << parameters[0..2]
+        if parameters[3]
+          # Option Name, Secret Name
+          @secret_options << [parameters[3], parameters[1]]
+        end
```

Comment:
You can make this a secret and a secret option at the same time? What does that look like?

Reply:
See the example openc3-cosmos-mqtt-test plugin I added.